adapter/src/main/java/com/twilio/kudu/dataloader/generator/TimestampGenerator.java

```diff
@@ -28,16 +28,20 @@ private TimestampGenerator() {
   }
 
   public TimestampGenerator(final long numDaysBefore) {
-    super(System.currentTimeMillis() - numDaysBefore * DateTimeUtils.MILLIS_PER_DAY, System.currentTimeMillis());
+    this.numDaysBefore = numDaysBefore;
```

Comment:
Can you explain this change, it looks weird.

Reply:
reverted this change